src/ContainerAppHelper.ts

```diff
@@ -0,0 +1,434 @@
+import * as os from 'os';
+import { Utility } from './Utility';
+import { GitHubActionsToolHelper } from './GitHubActionsToolHelper'
+import fs = require('fs');
+
+const ORYX_CLI_IMAGE: string = 'mcr.microsoft.com/oryx/cli:builder-debian-buster-20230208.1';
+const ORYX_BUILDER_IMAGE: string = 'mcr.microsoft.com/oryx/builder:20230208.1';
+const IS_WINDOWS_AGENT: boolean = os.platform() == 'win32';
+const PACK_CMD: string = IS_WINDOWS_AGENT ? path.join(os.tmpdir(), 'pack') : 'pack';
+const toolHelper = new GitHubActionsToolHelper();
+const util = new Utility();
+
+export class ContainerAppHelper {
+    readonly disableTelemetry: boolean = false;
+
+    constructor(disableTelemetry: boolean) {
+        this.disableTelemetry = disableTelemetry;
+    }
+
+    /**
+     * Creates an Azure Container App based from an image that was previously built.
+     * @param containerAppName - the name of the Container App
+     * @param resourceGroup - the resource group that the Container App is found in
+     * @param environment - the Container App Environment that will be associated with the Container App
+     * @param imageToDeploy - the name of the runnable application image that the Container App will be based from
+     * @param optionalCmdArgs - a set of optional command line arguments
+     */
+    public async createContainerApp(
+        containerAppName: string,
+        resourceGroup: string,
+        environment: string,
+        imageToDeploy: string,
+        optionalCmdArgs: string[]) {
+        toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}" based from image "${imageToDeploy}"`);
+        try {
+            let command = `az containerapp create -n ${containerAppName} -g ${resourceGroup} -i ${imageToDeploy} --environment ${environment}`;
+            optionalCmdArgs.forEach(function (val: string) {
+                command += ` ${val}`;
+            });
+            await util.executeAndThrowIfError(command);
+        } catch (err) {
+            toolHelper.writeError(err.message);
+            throw err;
+        }
+    }
+
+    /**
+     * Creates an Azure Container App based from a YAML configuration file.
+     * @param containerAppName - the name of the Container App
+     * @param resourceGroup - the resource group that the Container App is found in
+     * @param yamlConfigPath - the path to the YAML configuration file that the Container App properties will be based from
+     */
+    public async createContainerAppFromYaml(
+        containerAppName: string,
+        resourceGroup: string,
+        yamlConfigPath: string) {
+        toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}" from provided YAML "${yamlConfigPath}"`);
+        try {
+            let command = `az containerapp create -n ${containerAppName} -g ${resourceGroup} --yaml ${yamlConfigPath}`;
```

Comment:
We need to add `--output none` to the end of this command call per the latest changes made in `v1`

Reply:
Updated.